xbmc/addons/gui/skin/test/TestSkinTimers.cpp

```diff
@@ -24,12 +24,12 @@ TEST_F(TestSkinTimers, TestSkinTimerParsing)
   CSkinTimerManager skinTimerManager{infoMgr};
   skinTimerManager.LoadTimers(timersFile);
   // ensure only 5 timers are loaded (there are two invalid timers in the test file)
```

Comment:
Please update the annotations as well, got confused for a moment as there was 5 and changed to 6 below.

Not tested but to become the old TinyXML more and more away is always good :+1:.

Reply:
thanks